docs/examples.rst

```diff
@@ -195,6 +195,54 @@ the single dot notation:
     $ tmt run test --name .
 
 
+Create Tests
+------------------------------------------------------------------
+
+Use ``tmt tests create`` to create a new test. You need to specify
+the template using the ``--template`` option. You can also specify the test
+metadata template using ``--metadata-template``. If only ``--template``
```

Comment:
@therazix what is the difference between a template and metadata template? could we explain it better maybe?

Reply:
Test template is the test script itself (`test.sh`). Test metadata template is the `.fmf` file containing attributes like `test`, `duration`, etc. I will try to make it more clear in the documentation.

Reply:
thanks, that is sufficient for me